engine-precompiles/src/bn128.rs

```diff
@@ -353,8 +354,12 @@ impl<HF: HardFork> Bn128Pair<HF> {
 
 impl Precompile for Bn128Pair<Byzantium> {
     fn required_gas(input: &[u8]) -> Result<EthGas, ExitError> {
+        let input_len = u64::try_from(input.len())
+            .map_err(|_e| ExitError::Other(Borrowed("ERR_USIZE_CONVERSION")))?;
```

Comment:
I wonder if it makes sense to define a function like

```rust
const fn err_usize_conv<E: Error>(_e: E) -> ExitError {
  ExitError::Other(Borrowed("ERR_USIZE_CONVERSION"))
}
```

Then all these `try_from` calls can look like

```rust
let input_len = u64::try_from(input.len()).map_err(err_usize_conv)?;
```

I think that's a little nicer because (a) it doesn't repeat the `ERR_USIZE_CONVERSION` string multiple times, and (b) it's easier to read in my opinion.

Reply:
Was going to be going back over this and cleaning it up, just wanted the barebones in there first so that I knew how much of these I would be dealing with. That is a good suggestion though.